From 3d3ebca0f0bd58fac3900b001100eefa5fdb9c1e Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Fri, 21 Sep 2012 09:30:54 +0800
Subject: [PATCH 059/602] ENGR00215174 add legacy camera into built list.

the camera apk is updating by google.
pick up the legacy camera apk, because
new camera apk has not been ready.

Change-Id: Ia773b2673b66ded92443b20d7c0fb1586a82e614
Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 imx6/imx6.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index 69dcafb..8ff1bfd 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -15,6 +15,7 @@ PRODUCT_PACKAGES += \
 	Gallery		    			\
 	SoundRecorder				\
         Camera                                  \
+        LegacyCamera                            \
 	FSLOta					\
 	VideoEditor				\
 	FSLProfileApp				\
-- 
1.8.0

